README.md

```diff
@@ -39,6 +39,23 @@ Couch Potato is hosted as a gem which you can install like this:
 
     gem install couch_potato
 
+### Now using Ruby refinements
```

Comment:
ok - no prob. But don't you think at least an info about this should be added. Like 

'When using an older version than Ruby 2.0 please use CouchPotato 1.4.0' 

?